pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = -rsxX --reuse-db --tb=native
```

Comment:
can you expand the parameters here with their full name so they are less cryptic for future maintainers?